schema/course.schema.json

```diff
@@ -12,24 +12,31 @@
           "type": "object",
           "default": {},
           "properties": {
-            "_close": {
+            "_extensions": {
               "type": "object",
               "default": {},
               "properties": {
-                "closeButton": {
-                  "type": "string",
-                  "title": "Close button ARIA label",
-                  "default": "Select here to close the window.",
-                  "_adapt": {
-                    "translatable": true
+                "_close": {
+                  "type": "object",
+                  "title": "Close",
+                  "default": {},
+                  "properties": {
+                    "closeButton": {
+                      "type": "string",
+                      "title": "Close button ARIA label",
+                      "default": "Select here to close the window.",
+                      "_adapt": {
+                        "translatable": true
+                      }
+                    },
+                    "_navOrder": {
+                      "type": "number",
+                      "title": "Close button Nav order",
+                      "description": "Determines the order in which the object is displayed in the navigation bar. Negative numbers (e.g. -100) are left-aligned. Positive numbers (e.g. 100) are right-aligned.",
+                      "default": 150,
+                      "validators": []
```

Comment:
Comparing `_navOrder` config with other schemas, is `"validators": []` required?

Note, in the similar navigationLogo PR we have [`"_backboneForms": "Number"`](https://github.com/cgkineo/adapt-navigationLogo/blob/e4667a604d1a6e980a6c4ad3431c6df4bb639e75/schema/course.schema.json#L29) but in [core schemas](https://github.com/adaptlearning/adapt-contrib-core/blob/6ef84972f04cb760fb2862b0ba44990aeca73a24/schema/course.schema.json#L316) we don't have either `_backboneForms` or `validators`.

I know this PR is just for the schema nesting so I'm just querying here and I can raise a new issue if any amends needed?

Reply:
Empty validators can definitely be removed. 

With _backboneForms I've searched all the repos I have locally and the only schemas that have Number is navigationLogo and navigationTitle, for consistency I'll raise and remove from both of those repos.